src/towncrier/test/test_build.py

```diff
@@ -500,54 +501,92 @@ def test_no_package_changelog(self):
             ).lstrip(),
         )
 
-    def test_single_file(self):
+    def test_single_file_false(self):
         """
-        Enabling the single file mode will write the changelog to a filename
-        that is formatted from the filename args.
+        Disabling the single file mode will write the changelog to a filename
+        that is formatted from the filename args, w.r.t. [documentation]
+        (https://towncrier.readthedocs.io/en/latest/index.html?highlight=single_file#further-options)
+        as:
+            single_file = true  # if false, filename is formatted like `title_format`.
         """
         runner = CliRunner()
 
-        with runner.isolated_filesystem():
-            with open("pyproject.toml", "w") as f:
-                f.write(
-                    '[tool.towncrier]\n single_file=true\n filename="{version}-notes.rst"'
-                )
-            os.mkdir("newsfragments")
-            with open("newsfragments/123.feature", "w") as f:
-                f.write("Adds levitation")
+        def do_build_once_with(version, fragment_file, fragment):
+            with open(f"newsfragments/{fragment_file}", "w") as f:
+                f.write(fragment)
 
             result = runner.invoke(
                 _main,
                 [
                     "--version",
-                    "7.8.9",
+                    version,
                     "--name",
                     "foo",
                     "--date",
                     "01-01-2001",
                     "--yes",
                 ],
             )
+            # not git repository, manually remove fragment file
+            Path(f"newsfragments/{fragment_file}").unlink()
+            return result
 
-            self.assertEqual(0, result.exit_code, result.output)
+        results = []
+        with runner.isolated_filesystem():
+            with open("pyproject.toml", "w") as f:
+                f.write(
+                    '[tool.towncrier]\n single_file=false\n filename="{version}-notes.rst"'
+                )
```

Comment:
A bit easier to read.
```suggestion
                f.write(
                    '[tool.towncrier]\n'
                    ' single_file=false\n'
                    ' filename="{version}-notes.rst"\n'
                    ''
                )
```

Reply:
It is really tedious 😂